scenarios/preset/preset_grpc.py

```diff
@@ -33,7 +33,7 @@
 def main():
     container_list = []
     objects_struct = []
-    payload_filepath = '/tmp/data_file'
+    payload_filepath = '/tmp/data_file_' + args.size + 'k'
```

Comment:
i guess the problem you saw here was caused by a general mistake: this script was not expected to be used simultaneously. if we are trying to solve it, can we add some more unique data to the name (e.g. time?). if you are already running it for different sizes in parallel, i will not be surprised if some time in the future somebody will try to run it for two different networks or for two different object sets but with the same size, etc

Reply:
If the size is correct, then I don't see a problem with it.
To solve the problem with rights, I moved this file to the home directory.